src/GovUk.Education.ExploreEducationStatistics.Content.Requests/GovUk.Education.ExploreEducationStatistics.Content.Requests.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
This enables nullable for the entire subproject, so we can just get rid of `#nullable enable` from the request.

Reply:
Good spot